docs/src/main/asciidoc/redis.adoc

```diff
@@ -526,4 +526,4 @@ Once the build is finished, you can run the executable with:
 
 == Going further
 
-To learn more about the Quarkus Redis extension, check link:../redis-reference.adoc[the redis extension reference guide].
+To learn more about the Quarkus Redis extension, check link:../guides/redis-reference.adoc[the redis extension reference guide].
```

Comment:
Nice catch, it should be:
```suggestion
To learn more about the Quarkus Redis extension, check link:redis-reference.adoc[the Redis extension reference guide].
```

Reply:
Thank you for your advice !  Exactly it should be link:redis-reference.adoc[the Redis extension reference guide].